citations-tool/tool/src/webapp/js/new_resource.js

```diff
@@ -786,5 +792,93 @@ citations_new_resource.init = function() {
 $(document).ready(function(){
 	citations_new_resource.init();
 	citations_new_resource.setupToggleAreas('toggleAnchor', 'toggledContent', false, 'fast');
-});
 
+
+    // add ck editor and event handlers to existing sections
+    CKEDITOR.disableAutoInline = true;
+
+    $( "div[id^='sectionInlineEditor']" ).each(function( index ) {
+        CKEDITOR.instances[this.id].on( 'blur', function( evt ) {
+
+            var actionUrl = $('#newCitationListForm').attr('action');
+            $('#citation_action').val('add_section');
+            var params = $('#newCitationListForm').serializeArray();
+            params.push({name:'addSectionHTML', value:$(evt.editor.getData()).html()});
+            params.push({name:'locationId', value:evt.editor.name.substring('sectionInlineEditor'.length, evt.editor.name.length)});
+
+            $.ajax({
+                type		: 'POST',
```

Comment:
Indenting inside the line with tabs leads to uneven indentation when the tab size is different.